_pages/about-us/teams/engineering.md

```diff
@@ -148,7 +137,7 @@ Most of the Engineering team works on a mix of partner agency projects and inter
 
   - [18F-dev@gsa.gov](https://groups.google.com/a/gsa.gov/forum/#!forum/18fdev). All new engineering chapter members should be added by their facilitator. If for some reason you didn't get automatically added, give a shout in [#eng-leadership](https://gsa-tts.slack.com/messages/eng-leadership). Someone there will get you set up.
   - [18f-eng-leadership@gsa.gov](https://groups.google.com/a/gsa.gov/forum/#!forum/18f-eng-leadership). This is a good way to write to the entier engineering leadership team at once.
-  - Optional listservs that may be of interest: [Interagency Open Data Working Group](/general-contacts-and-listservs/#interagency-open-data-working-group), [Security Today](/general-contacts-and-listservs/#security-today).
+  - Optional listserv that may be of interest: [Interagency Open Data Working Group](/general-contacts-and-listservs/#interagency-open-data-working-group)
```

Comment:
This seems like a good time to correct "entier" above.

Reply:
Er, yeah — missed that typo! I'll grab it. Going to save any further major revisions to this page for someone else, though — it needs a larger overhaul, and I'm not the right person for it.